participant-manager/src/app/site-coordinator/sites/import-email-list/import-email-list.component.ts

```diff
@@ -69,9 +67,8 @@ export class ImportEmailListComponent extends UnsubscribeOnDestroyAdapter {
                 ` emails failed to import.</br>` +
                 `Reason for import failure for these could be one of the following:<br/>
                 1.Email not in proper format <br/>
```

Comment:
Not sure how this gets rendered - but is a space needed between the number e.g. "1." and the text e.g. "Email"

Reply:
Updated.